gwsumm/plot/core.py

```diff
@@ -324,6 +324,8 @@ def pid(self):
             filts = "".join(map(str,
                 [getattr(c, 'filter', getattr(c, 'frequency_response', ''))
                  for c in self.channels]))
+            if self.filterstr:
+                filts = "".join([filts,self.filterstr])
```

Comment:
the `filterstr` will only be defined for trigger plots, this line will/should fail for other types. I think this method needs to be subclassed for trigger-type plots to correctly. do what is being done here.